### PR TITLE
Add python-neo

### PR DESCRIPTION
**Pull request by jakirkham**
Adds a recipe to build `neo`. Prefixed with `python-` due to it being a Python package. Generated with `conda skeleton pypi` and cleaned up for addition.

### CODE REVIEW

**Comment by jakirkham** [comment]
Ready for review (and merge).

File: recipes/python-neo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - neo
+    - neo.core
+    - neo.io
+    - neo.test
+    - neo.test.iotest
+
+about:
+  home: http://neuralensemble.org/neo
+  license: BSD 3-Clause
+  summary: Neo is a package for representing electrophysiology data in Python, together with support for reading a wide range of neurophysiology file formats.
```
Comment (msarahan) [inline recipes/python-neo/meta.yaml]: This is probably too long.  Can you try to keep to 80 characters for the summary?  We didn't get to that in the meeting on Friday, but that's one of our requirements for metadata, ensuring that your package will play nicely with Navigator.

Comment (msarahan) [inline recipes/python-neo/meta.yaml]: (there's also the description field where you can be more long-winded)

Comment (jakirkham) [inline recipes/python-neo/meta.yaml]: Yeah, was also thinking this is too long, but it had come from their [description]( https://github.com/NeuralEnsemble/python-neo/blob/3f9a5a62fda9b3f4d19a51f44b037338cc55d21a/setup.py#L22 ). Will take some liberties with shortening it.

Comment (jakirkham) [inline recipes/python-neo/meta.yaml]: Rephrased so it is shorter. We should discuss these precise restrictions in the next meeting, but keep the `description` short generally seems reasonable to me.

**Comment by msarahan** [comment]
LGTM.  Merging.

**Comment by jakirkham** [comment]
Thanks @msarahan.

**Comment by jakirkham** [comment]
@apdavison @samuelgarcia, just letting you know that we have added `python-neo` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) ( [`conda install -c conda-forge python-neo`]( http://anaconda.org/conda-forge/python-neo ) ). The maintenance burden here is light as all releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this feedstock (repo)]( https://github.com/conda-forge/python-neo-feedstock ). If any of you have any interest in being added as a maintainer (to make new releases for instance), we would be more than happy to add any of you to the maintainers list.

**Comment by samuelgarcia** [comment]
Thank you very much for this.

Samuel

**Comment by apdavison** [comment]
Great!

**Comment by jakirkham** [comment]
Of course, if either of you have any interest in being added as maintainers, please let me know. I would be more than happy to add you. Basically, this would give you commit privileges on the repo, which could be useful when adding new releases for instance.

**Comment by apdavison** [comment]
Yes please, it would be useful if myself and @samuelgarcia were maintainers.

**Comment by jakirkham** [comment]
Excellent. This PR ( https://github.com/conda-forge/python-neo-feedstock/pull/1 ) will do exactly that. A description of how this works is included in the PR.

**Pull request merged**